README.md

```diff
@@ -56,7 +56,7 @@ Unlike the public demo, the playground sessions are deleted after 4 hours. Apart
 **_Portainer_** has full support for the following Docker versions:
 
 * Docker 1.10 to the latest version
-* Docker Swarm >= 1.2.3
+* Docker Swarm >= 1.2.3 (support for standalone Docker Swarm is dropped in Portainer 1.17.0, but the built-in Swarm Mode is fully supported)
```

Comment:
Sounds a bit contradictory to me, it says that we support Docker Swarm >= 1.2.3 but that the support is dropped...

I believe that we shoud just remove it.

Reply:
I actually removed that line first, but then realized, we should keep a note for a while that the support was dropped in 1.17.0, though prior versions of Portainer (such as 1.16.x) might still support standalone Docker Swarm versions >= 1.2.3. After a while when very few people are using 1.16.x, we can perhaps remove it completely.

Reply:
However, if the intent of README is to reflect the most recent state of the software then I can go ahead and remove this line.

Reply:
Agree, just need to rephrase it

Reply:
* Standalone Docker Swarm >= 1.2.3 _(**NOTE:** Use of Standalone Docker Swarm is being discouraged since the introduction of built-in Swarm Mode in Docker. While older versions of Portainer had support for Standalone Docker Swarm, Portainer 1.17.0 and newer will not support it.  However, the built-in Swarm Mode of Docker is fully supported.)_

Reply:
Will something like this work?

Reply:
@ibnesayeed looks good!